uqcsbot/holidays.py

```diff
@@ -94,14 +76,45 @@ def get_holidays_from_csv() -> List[Holiday]:
 
     return holidays
 
-
-def get_holiday_page() -> bytes:
+def get_holiday_page() -> bytes | None:
     """
     Gets the holiday page HTML
     """
     try:
         response = requests.get(HOLIDAY_URL)
         return response.content
     except RequestException as e:
-        bot.logger.error(e.response.content)
+        logging.warning(e.response.content)
     return None
+
+class Holidays(commands.Cog):
+    def __init__(self, bot: UQCSBot):
+        self.bot = bot
+        self.bot.schedule_task(self.holiday, trigger='cron', hour=16, minute=12, timezone='Australia/Brisbane')
```

Comment:
Might need to actually be 9am

```suggestion
        self.bot.schedule_task(self.holiday, trigger='cron', hour=9, minute=0, timezone='Australia/Brisbane')
```

Reply:
Ah that is a big oopsies, thanks